From linux-kernel-owner+greg=40kroah.com-S1751083AbVI1QF6@vger.kernel.org Thu Sep 29 05:26:19 2005
Date: Wed, 28 Sep 2005 18:06:04 +0200
From: Borislav Petkov <bbpetkov@yahoo.de>
Subject: USB: make usb storage note visible in Kconfig
Message-ID: <20050928160604.GA23955@gollum.tnic>


the following patch splits the NOTE: in the Device Drivers->USB submenu of
Kconfig thus making the whole of it readable on 600x800 terminals.
(Otherwise, the line was too big and disappeared into nowhere.)

	
Signed-off-by: Borislav Petkov <petkov@uni-muenster.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/storage/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

---
--- gregkh-2.6.orig/drivers/usb/storage/Kconfig
+++ gregkh-2.6/drivers/usb/storage/Kconfig
@@ -2,7 +2,8 @@
 # USB Storage driver configuration
 #
 
-comment "NOTE: USB_STORAGE enables SCSI, and 'SCSI disk support' may also be needed; see USB_STORAGE Help for more information"
+comment "NOTE: USB_STORAGE enables SCSI, and 'SCSI disk support'"
+comment "may also be needed; see USB_STORAGE Help for more information"
 	depends on USB
 
 config USB_STORAGE
